src/shared/questions/links.js

```diff
@@ -102,18 +130,30 @@ async function updatelinks(links) {
         message: "Do you want to update another link?",
       },
     ]);
+    
+    if(answers.color !== "") {
+      links[links.length - 1].color = answers.color;
+    }
+
+    if(answers.group !== "") {
+      links[links.length - 1].group = answers.group;
+    }
+
     choiceLinks.map((link) => {
       if (link.name === answers.link) {
         link.name = name;
         link.url = url;
         link.icon = selectedIcon;
+        link.color = color;
+        link.group = group;
       }
     });
+
     if (!updateLink) {
       let result = [];
       choiceLinks.map((link) => {
-        const { name, url, icon } = link;
-        result.push({ name, url, icon });
+        const { name, url, icon, color, group } = link;
```

Comment:
You need to check below if the `color` and group are `empty` or now then add in the `result` array.

Reply:
Could you clarify some more, I am not able to understand, sorry.

Reply:
Here are directly pushing to get the values from the link after that we need to check the `color` and `group` are not empty then push to the result array It will be wrong if we directly push here ` result.push({ name, URL, icon, color, group });`